Mimir/src/models/PlayerStat.php

```diff
@@ -200,6 +200,92 @@ protected function _getPlacesSummary(int $playerId, array $games)
         ]);
     }
 
+    /**
+     * @param \Common\Ruleset $rules
+     * @param int[] $currentScores
+     * @param int $han
+     * @param int|null $fu
+     * @param int $winnerId
+     * @param int|null $loserId
+     * @param int[] $riichiIds
+     * @param int $playerId
+     * @param int $dealerId
+     * @param int $honba
+     * @param int $riichiBetsCount
+     * @param bool $isTsumo
+     * @param int|null $paoPlayerId
+     * @param int|null $closestWinnerForMultiron
+     * @param int $totalRiichiInRoundForMultiron
+     * @return int
+     * @throws InvalidParametersException
+     */
+    protected function _getPointsDelta(\Common\Ruleset $rules, array $currentScores, int $han, ?int $fu, int $winnerId, ?int $loserId, array $riichiIds, int $playerId, int $dealerId, int $honba, int $riichiBetsCount, bool $isTsumo, ?int $paoPlayerId, ?int $closestWinnerForMultiron = null, int $totalRiichiInRoundForMultiron = 0)
```

Comment:
Could you please remove type hints for now? In general, it's not guaranteed that input types would match expected ones, at least for now, and I don't think we want to ensure strict typing across all php codebase in pantheon. The bad thing is, when actual input type doesn't match the expected one, the whole application just fails with an error. So I think having types in phpdoc is more than enough now.

Reply:
Also I'd ask to separate args with line breaks. There's too many of them, so multiline list would be easier to read

Reply:
Done